wordpress-mobile/WordPress-iOS
# Don’t fail post upload when only attachment of media failed.

Fixes #8727 

This PR just does a small change, to the postUpload method in order to not make it fail when an attachment fails.

To test:
 - See the ticket with the step on how to reproduce the original crash.

cc @diegoreymendez i think the reason was making the app crash after, was that when post upload was successful before we attached media to it, we updating and deleting the post.revision and probably made the Post entity referred by AztecPostViewController to be delete from the MOC and made the MOC nil on the post.
